recipe/meta.yaml

```diff
@@ -32,12 +32,15 @@ requirements:
     - pydantic
     - pynacl
     - python-keycloak
```

Comment:
```suggestion
    - python-keycloak >=2.5.0,<2.6.0
```

Reply:
This needs to be added to force the pin you wanted, unfortunately the smithy doesn't seem to find the correct version of `requests-toolbet` from within conda-forge

Reply:
disregard this comment, as it seems that pinning Keycloak alone didn't fix the issue